api/v1alpha1/common_webhook.go

```diff
@@ -173,6 +174,18 @@ func (f *FloatStr) Default(root interface{}, field *reflect.StructField) {
 	}
 }
 
+func (in *LabelSelectorRequirements) Validate(root interface{}, path *field.Path) field.ErrorList {
```

Comment:
This looks a bit weird touching the common webhook but I trust you know what you're doing.  Is it just adding some missing validation?

Reply:
Yeah, just adding missing validations.

Without this validation I get an error during the apply phase that "`foo` is not a valid operation" rather than rejection during resource creation.

The same thing happens for `podchaos`, for example, if using expression selectors. This does seem like a small self-contained PR upstream separate from these changes maybe, given it has wider reach?